packages/common/random-access-storage/project.json

```diff
@@ -12,7 +12,17 @@
       }
     },
     "lint": {},
-    "test": {}
+    "test": {
+      "executor": "@nx/vite:test",
+      "inputs": [
```

Comment:
you need to add a glob for source files here as well - see the definition for `test` in root `nx.json`

Reply:
yeah, beware that it will inherit all options from nx.json so may want to explicitly define a lot of them for now until we move everything over to vitest in order to avoid conflicts between options of the executors